### DISCUSSION TITLE
有人生环 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#428] [General]
# 有人生环 - Miao Yu | 于淼

https://yufree.cn/cn/2023/08/03/805/

**Comment by DayuGuo** [comment]
之前有看到过一个AMR相关的桌游项目，作为教学互动的一环 <https://ce4amr.leeds.ac.uk/projects/follow-on-funding-tackling-amr-in-schools-nepal/amr-playing-cards/> 。
这种有特色有主题的桌游项目都挺有趣的~  有机会我也找同学玩玩您这款！

**Reply by yufree** [reply]
欢迎试玩！应该是两副扑克牌就够了，体力牌可能要另算。另外我稍微改了下回复里链接，这个设计也很有意思，现在有AI设计主题科普的游戏应该方便很多了。
